modules/core/shared/src/main/scala/lucuma/core/enums/ProposalClass.scala

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.enums
+
+import lucuma.core.util.Enumerated
+
+sealed abstract class ProposalClass(val label: String) extends Product with Serializable
```

Comment:
How is this new? I thought it would be present on the scala 2 version

Reply:
Thanks. Seems it was deleted in https://github.com/gemini-hlsw/lucuma-core/pull/523. Must be a bad merge.